README.md

```diff
@@ -86,16 +86,17 @@ stage("Security Scan") {
                 blackDuckPrComment = true
             }
 
-            synopsys_scan scan_type: "BLACKDUCK", blackduck_url: "${env.BLACKDUCK_URL}", blackduck_api_token: "${env.BLACKDUCK_TOKEN}", blackduck_scan_full: "${blackDuckScanFull}", blackduck_automation_prcomment: "${blackDuckPrComment}"
+            synopsys_scan scan_type: "BLACKDUCK", bridge_blackduck_url: "https://example.com", bridge_blackduck_api_token: "YOUR_BLACKDUCK_TOKEN", 
+                    bridge_blackduck_scan_full: "${blackDuckScanFull}", bridge_blackduck_automation_prcomment: "${blackDuckPrComment}"
         }
     }
 }
 ```
-Make sure to provide the required parameters such as `blackduck_url` and `blackduck_api_token` with the appropriate values.
+Make sure to provide the required parameters such as `bridge_blackduck_url` and `bridge_blackduck_api_token` with the appropriate values.
 
 Or the values are configure in Jenkins Global Configuration, then the below example can be used - 
```

Comment:
Maybe this line can be modified as below:
If the values are configured in **Jenkins Global Configuration**, you can use the following example.